Testing/Map.gd

```diff
@@ -0,0 +1,34 @@
+# This is a class for maps and all data related to them - namely terrain
```

Comment:
I don't see anything in the best practices but it did take me a while to find this code in the IDE since this is under `Testing`, not under `Scripts`. If it's not against some practice I vote we consolidate this code into the scripts folder where the other scripts are, like `Scripts/maps/` or something.

Reply:
Moved